### DISCUSSION TITLE
Did not search for ...

### DISCUSSION THREAD

**Discussion by pummra** [cross-seed/cross-seed#99] [Q&A] [answered]
Got everything up and running with the script now and really happy with the way it works on my seedbox. I have daemon mode up and running in a screen instance. 

Just had a look at the screen instance to check everything is working fine, which it looks as though it is. However some torrents that are pushed in come back with a message: 

```
info: [server] Received name <xxxx>
info: [server] Did not search for <xxxx>
```

<xxxx> is what I have put in place of the torrent name. What makes the script decide if it will search or not? I know for a fact there is cross-seed opportunities on that torrent.

**Comment by mmgoodnow** [comment, marked as answer]
It means that it got filtered by its content, which means it's either a single episode torrent or not all contents are video files.

Here's a more thorough explanation of the content filtering: https://github.com/mmgoodnow/cross-seed/discussions/91#discussioncomment-895926
